packages/components/src/components/tabs/tabs.jsx

```diff
@@ -119,47 +130,61 @@ const Tabs = ({
                         'dc-tabs__list--center': center,
                         'dc-tabs__list--header-fit-content': header_fit_content,
                         [`dc-tabs__list--${className}`]: className,
+                        'dc-tabs__list--overflow-hidden': is_scrollable,
                     })}
                     ref={tabs_wrapper_ref}
                 >
-                    {React.Children.map(children, (child, index) => {
-                        if (!child) return null;
-                        const { count, header_content, icon, label, id } = child.props;
-                        return (
-                            <Tab
-                                active_icon_color={active_icon_color}
-                                className={className}
-                                count={count}
-                                icon={icon}
-                                icon_color={icon_color}
-                                icon_size={icon_size}
-                                is_active={index === active_tab_index}
-                                key={label}
-                                is_label_hidden={children.length === 1 && single_tab_has_no_label}
-                                label={label}
-                                id={id}
-                                top={top}
-                                bottom={bottom}
-                                header_fit_content={header_fit_content}
-                                active_tab_ref={index === active_tab_index ? active_tab_ref : null}
-                                header_content={header_content}
-                                onClick={() => onClickTabItem(index)}
-                                setActiveLineStyle={setActiveLineStyle}
+                    <ConditionalWrapper
+                        condition={is_scrollable}
+                        wrap={content => (
+                            <ThemedScrollbars
```

Comment:
The `ThemedScrollbars` component takes an `is_bypassed` prop, could it be useful here? e.g.

```JS
<ThemedScrollbars is_bypassed={is_scrollable}>
    { /* etc. */ }
</ThemedScrollbars>
```

Reply:
Of course, why the hell I needed ConditionalWrapper, ahh

thank you!